scripts/update-pa11y-config.js

```diff
@@ -11,72 +11,54 @@
  * the visual regression tests. (.pa11yci.ci.json)
  *
  * Options for both configs can be added into the baseConfig object.
+ * @ TODO: Upgrade storybook to 6 and use JSON export
```

Comment:
Storybook is v6 isnt it?

Reply:
This TODO was already here, I just moved it. We are on v6 so whilst we could probably do this now it was beyond the scope of this PR.